modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/radio/RadioDDMFormFieldValueRendererTest.java

```diff
@@ -62,15 +62,15 @@ public void testRender() throws Exception {
 
 		DDMFormFieldValue ddmFormFieldValue =
 			DDMFormValuesTestUtil.createDDMFormFieldValue(
-				"Radio", new UnlocalizedValue("value 1"));
+				"Radio", new UnlocalizedValue("option with &, option 2"));
```

Comment:
@leticia-maciel you must pass the option value here (`new UnlocalizedValue("value 1"))`) instead of the option label. Also, you must pass only one value, because you can only choose one option in a Radio field.